### PR TITLE
bitopro: update rl

### PR DESCRIPTION
**Pull request by sc0Vu**

### CODE REVIEW

File: ts/src/bitopro.ts [hunk 1]
```diff
@@ -136,19 +136,19 @@ export default class bitopro extends Exchange {
...
-                    ],
-                    'put': [
-                        'orders',
-                    ],
-                    'delete': [
-                        'orders/{pair}/{id}',
-                        'orders/all',
-                        'orders/{pair}',
-                    ],
+                    'post': {
```
Comment (carlosmiei) [inline ts/src/bitopro.ts:139]: @sc0Vu what about the other endpoints? can't we add the weight to them as well?

Comment (sc0Vu) [reply ts/src/bitopro.ts:139]: @carlosmiei it seems they didn't add rate limit for all endpoints. I'll check with them.

Comment (sc0Vu) [reply ts/src/bitopro.ts:139]: They replied in the channel. I'll update.

Open API:
 ◦ 600 requests per minute per IP

Auth API:
 ◦ 600 requests per minute per IP
 ◦ 600 requests per minute per user account

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**